docker-compose/cassandra-4.0/.env

```diff
@@ -0,0 +1,3 @@
+COMPOSE_PROJECT_NAME=cass40-stargate
+CASSTAG=4.0
```

Comment:
should we also fix this to the version we currently support `4.0.1`?

Reply:
fixing